Source/diablo_msg.cpp

```diff
@@ -1,23 +1,17 @@
-/**
- * @file error.cpp

```

Comment:
This is needed for doxygen to pick up the file when generating documentation

Reply:
Do we want to generate documentation for implementation files?

Reply:
I think so, there is a lot of details that happens in the internals. Also since it's not a library I think the documentation should be targeted at internal use as well.

Reply:
Anyway, done